design/sketch.webidl

```diff
@@ -13,7 +13,7 @@ enum WebGPULogEntryType {
 
 interface WebGPULogEntry {
     readonly attribute WebGPULogEntryType type;
-    readonly attribute any object;
+    readonly attribute any data;
```

Comment:
nit: this is the object on which the error happened so data is a bit too generic. Right now it looks like any userdata.

Reply:
using `object` there isn't appreciated by W3C validator

Reply:
Oh :(

Reply:
+1 to sourceObject